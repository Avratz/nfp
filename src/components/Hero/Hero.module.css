.hero {
  text-align: center;
  margin-top: 4rem;
}

.toptitle {
  font-weight: var(--font-bold) !important;
  font-family: var(--font-poppins) !important;
  color: var(--neutral-5);
  text-transform: uppercase;
  font-size: var(--text-2xs);
  line-height: var(--line-2xs);
  animation: var(--fadeInLong);
}

.title {
  color: var(--neutral-8);
  letter-spacing: -0.5px;
  margin-top: 0.5rem;
  margin-bottom: 1.5rem;
  margin-left: auto;
  margin-right: auto;
  animation: var(--fadeInLong);
}

.exploreButton {
  border-color: var(--neutral-4);
  animation: var(--fadeInLong);
}

@media screen and (min-width: 768px) {
  .hero {
    margin-top: 8rem;
  }
}

@media screen and (min-width: 1200px) {
  .hero {
    padding-top: 6rem;
    padding-bottom: 6rem;
  }
  .toptitle {
    font-size: var(--text-xs) !important;
  }
  .title {
    font-size: var(--text-5xl) !important;
  }
}

@media screen and (min-width: 1920px) {
  .hero {
    padding-top: 8rem;
    padding-bottom: 8rem;
  }
}
