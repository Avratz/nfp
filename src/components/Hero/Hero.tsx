import { Button } from '@/ui/Button/Button'
import { Container } from '@/ui/Container/Container'
import { Heading } from '@/ui/Heading/Heading'

import styles from './Hero.module.css'

export function Hero() {
  return (
    <Container as="section" className={styles.hero}>
      <p className={styles.toptitle}>
        Create, explore, & SELL digital art NFTs.
      </p>
      <Heading as="h2" size="4xl" className={styles.title}>
        The new creative economy.
      </Heading>
      <Button href="/" className={styles.exploreButton}>
        Explore
      </Button>
    </Container>
  )
}
