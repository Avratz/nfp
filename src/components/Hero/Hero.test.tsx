import { render } from '@testing-library/react'
import { Hero } from './Hero'

describe('Hero', () => {
  it('renders Hero', () => {
    const { container } = render(<Hero />)
    expect(container).toMatchSnapshot()
  })
})
