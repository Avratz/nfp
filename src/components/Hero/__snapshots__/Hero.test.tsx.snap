// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hero renders Hero 1`] = `
<div>
  <section
    class="hero container containerPadding"
  >
    <p
      class="toptitle"
    >
      Create, explore, & SELL digital art NFTs.
    </p>
    <h2
      class="heading 4xl title"
    >
      The new creative economy.
    </h2>
    <a
      class="link button primary base exploreButton"
      href="/"
    >
      Explore
    </a>
  </section>
</div>
`;
